integration/kubernetes/alluxio-worker.yaml

```diff
@@ -0,0 +1,70 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: alluxio-worker
+spec:
+  template:
+    metadata:
+      labels:
+        name: alluxio-worker
+        app: alluxio-worker
+    spec:
+      hostNetwork: true
+      hostPID: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: alluxio-worker
+          image: alluxio/alluxio:1.6.1
+          resources:
+            requests:
+              cpu: "0.5"
+              memory: "1024M"
+            limits:
+              cpu: "1"
+              memory: "1024M"
+          command: ["/entrypoint.sh"]
+          args: ["worker"]
+          env:
+          - name: ALLUXIO_WORKER_HOSTNAME
+            valueFrom:
+              fieldRef:
+                fieldPath: status.hostIP
+          envFrom:
+          - configMapRef:
+              name: alluxio-common-config
+          - configMapRef:
+              name: alluxio-worker-config
+          ports:
+          - containerPort: 29998
+            name: rpc
+          - containerPort: 29999
+            name: data
+          - containerPort: 30001
+            name: web
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: alluxio-ramdisk
+              mountPath: /dev/shm
+            - name: alluxio-domain
+              mountPath: /opt/domain
+      restartPolicy: Always
+      volumes:
+        - name: alluxio-ramdisk
+          emptyDir:
+            medium: "Memory"
+            sizeLimit: "768M"
```

Comment:
why 768M?

Reply:
my bad, no good reason. what would be a good default since it is highly dependent on workload. does 2 GB sound like a reasonable minimum?

Reply:
sg